Sources/PointFree/Account/Team.swift

```diff
@@ -101,27 +99,22 @@ private let requireTeammate: MT<Tuple2<User.ID, User>, Tuple2<User, User>> = fil
   or: redirect(to: .account(), headersMiddleware: flash(.error, "Could not find that teammate."))
 )
 
-private func sendEmailsForTeammateRemoval(owner: User, teammate: User) -> Parallel<Prelude.Unit> {
+private func sendEmailsForTeammateRemoval(owner: User, teammate: User) async throws {
+  guard owner.id != teammate.id
+  else { return }
 
-  guard owner.id != teammate.id else {
-    return pure(unit)
-  }
+  async let response1 = sendEmail(
+    to: [teammate.email],
+    subject: "You have been removed from \(owner.displayName)’s Point-Free team",
+    content: inj2(youHaveBeenRemovedEmailView(.teamOwner(owner)))
+  )
 
-  return zip2(
-    parallel(
-      sendEmail(
```

Comment:
replacing a fire-and-forgot-zip-parallel with a few `async let`s.

Reply:
I think the one difference here is that `async let`s will still throw and cause the request to fail. That might be fine, but if we wanna stay on the happy path for the request maybe we should use `Task` here as well?

Reply:
yeah, I decided to just inline it all: https://github.com/pointfreeco/pointfreeco/pull/805/commits/95319eba064fdc4abe106583e1b5bbcb9ccff968